generators/exercises/say.py

```diff
@@ -0,0 +1,52 @@
+FUNC_PROTO = """\
+#include "vendor/unity.h"
+
+#include <stdint.h>
+
+#define BUFFER_SIZE 240
+
+extern void say(char *buffer, int64_t number);
+"""
+
+def extra_cases():
+    return [
+        {
+            "description": "additional big number",
+            "property": "say",
+            "input": {
+                "number": 19011016013
+            },
+            "expected": "nineteen billion eleven million sixteen thousand thirteen"
+        },
+        {
+            "description": "different big number",
+            "property": "say",
+            "input": {
+                "number": 812000070017
+            },
+            "expected": "eight hundred twelve billion seventy thousand seventeen"
+        },
+        {
+            "description": "alternative big number",
+            "property": "say",
+            "input": {
+                "number": 60010015018
+            },
+            "expected": "sixty billion ten million fifteen thousand eighteen"
+        }
+    ]
+
+def gen_func_body(prop, inp, expected):
+    number = inp["number"]
+    if expected.__class__ == dict:
```

Comment:
Something like this would be better in this case as well:
```suggestion
    if isinstance(expected, dict):
```